plugins/modem-manager/fu-mm-device.c

```diff
@@ -1681,6 +1691,11 @@ fu_mm_device_set_quirk_kv(FuDevice *device, const gchar *key, const gchar *value
 		return TRUE;
 	}
 
+	if (g_strcmp0(key, "FirehoseProgFile") == 0) {
```

Comment:
This needs to be `ModemManagerFirehoseProgFile` and also documented in the README.md file please. Then LGTM.